src/domains/pay-wallet-app/api/healthcheck/v1/_base_policy.xml.tpl

```diff
@@ -0,0 +1,71 @@
+<policies>
+    <inbound>
+        <base />
+
+        <send-request mode="new" response-variable-name="walletServiceLiveness" timeout="10">
```

Comment:
maybe here we can add an "ignore-error" set to true for all of this send-request to avoid that the whole request results in an error if one of those modules is down and then add a check against http error code too in the outbound policy section.
This way the response will successfully be returned even if a single service is down (not responding to healthcheck api call)

Reply:
correct!